src/cmd/uds.go

```diff
@@ -222,6 +228,74 @@ var logsCmd = &cobra.Command{
 	},
 }
 
+var uiCmd = &cobra.Command{
+	Use:     "ui",
+	Aliases: []string{"u"},
+	Short:   lang.CmdUIShort,
+	RunE: func(_ *cobra.Command, _ []string) error {
+
+		// Create a temporary directory to hold the embedded files
```

Comment:
what do you think of moving this code into its own `ui.go` or even in `vendored.go`?

Reply:
Yeah, I considered adding it to `vendored.go` but didn't think it really fit with the way we are vendoring the other tools. I'll go ahead and add it into its own `ui.go`